.github/workflows/benchmarks.yaml

```diff
@@ -14,8 +14,8 @@ jobs:
     name: Benchmarks
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
```

Comment:
`v4` is the latest major version of both of these actions at the time of this writing.